src/wallet/rpc/spend.cpp

```diff
@@ -71,9 +71,13 @@ static void InterpretFeeEstimationInstructions(const UniValue& conf_target, cons
     } else {
```

Comment:
nit:
iwyu
```cpp
#include <util/strencodings.h>
```

Reply:
Sorry, I don't understand what you mean. Do you want me to use one of the functions in `strencodings` to manipulate the `fee_rate` parameter before sending it to `options.pushKV`?

Reply:
I meant Import what you use.
`ToLower()` function is declared here.
https://github.com/bitcoin/bitcoin/blob/4ad83ef09b772f3f1e8ea9ea3a8c43dcd9db8458/src/util/strencodings.h#L327

Reply:
Ah, thanks. I'm very new to C++, so I didn't realize I could even use things that weren't imported. Didn't realize I should also include a header statement, since it compiled just fine! Added the include.